src/app/locationForm/common/FormEdit.jsx

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react';
```

Comment:
Whereas I think the Form container's logic is going to remain pretty standard across the different form screens, looking at the designs it seems like the presentation is actually going to be fundamentally different for nearly every screen (other than probably description and location name). So, I think we'll need the flexibility to specify them separately, which FormView and FormEdit can't really provide.
Of course, I don't think there's any harm in having FormView and FormEdit for convenience right now, before we build out the actual screens.

Reply:
Yes, I agree, most edit forms are going to be pretty custom, and different from other forms. Please see my solution in https://github.com/streetlives/streetlives-web/pull/8/commits/8723b4ce0c82ccd767c5542d324e568d3635d336#diff-c57e579bad4f4f74984ee84e76c54e7dR14 which is to create a custom LocationNumberEdit form.